addon/components/data-column.js

```diff
@@ -34,43 +34,41 @@ function copy(object) {
 }
 
 export default Component.extend({
-  tagName: 'td',
+  tagName: 'col',
   attributeBindings: ['style'],
 
   _property: computed('property', function() {
     return $.extend({}, defaultPropertyObject, get(this, 'property'));
   }),
-
-  style: computed('_values', '_property.style',
-  function() {
-    let values = get(this, '_values');
+  style: computed('_property.style', function() {
     let style  = copy(get(this, '_property.style'));
     let styles = [];
 
-    // if style property is a function, return style depending on value
+    // if style property is a function style will be handled in cell
+    // (data-cell.js)
     for (let property in style) {
       if (style.hasOwnProperty(property) &&
-          typeof style[property] === 'function') {
-        style[property] = style[property](...values);
-      }
-      if (typeof style[property] === 'boolean') {
-        style[property] = style[property] ? property : null;
-      }
-
-      let value = style[property];
-      let stylingProperty = STYLING_PROPERTIES[property];
-
-      if (value && stylingProperty) {
-        let key    = stylingProperty.key || dasherize(property);
-        let suffix = stylingProperty.suffix || '';
-
-        styles.push(`${key}: ${value}${suffix};`);
+          typeof style[property] !== 'function') {
+        if (typeof style[property] === 'boolean') {
+          style[property] = style[property] ? property : null;
+        }
+
+        let value = style[property];
+        let stylingProperty = STYLING_PROPERTIES[property];
+
+        if (value && stylingProperty) {
+          let key    = stylingProperty.key || dasherize(property);
+          let suffix = stylingProperty.suffix || '';
+          if (typeof value === 'string') {
+            suffix = '';
+          }
+          styles.push(`${key}: ${value}${suffix};`);
+        }
       }
     }
 
     return htmlSafe(styles.join(' '));
   }),
-
```

Comment:
I guess (I'm not sure) there is a lot of stuff in this copied file you don't need